src/main/java/frc/robot/subsystems/Intake/Intake.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.subsystems.Intake;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Robot;
+
+public class Intake extends SubsystemBase {
+    private final IntakeIO io;
+    private final IntakeInputsAutoLogged inputs;
+
+    private Intake(IntakeIO io) {
+        inputs = new IntakeInputsAutoLogged();
+        this.io = io;
+        if (Robot.isReal()){
+
+        }
+    }
```

Comment:
What about getInstance? You should pass the io parameter to the constructor in the getInstance, based on whether the robot is real or not. For example:
`
public Intake getInstance() {
    if (INSTANCE == null) {
        if (Robot.isReal) {
            INSTANCE = new Intake(new IntakeIOReal());
        } else {
            // Create INSTANCE with the sim io
        }
    }
}
`